config/defaults-config.json

```diff
@@ -108,5 +108,7 @@
   "CLEANUP_TELEGRAM": true,
   "CLEANUP_TIME_TG":"10",
   "CLEANUP_DATABASE": true,
-  "CLEANUP_TIME_DB":"10080"
+  "CLEANUP_TIME_DB":"10080",
```

Comment:
- If this is an integer it shouldn't have the quotes, but perhaps we should treat it as a string because:
- Having this value in seconds is overtly confusing for users (need to do math to realize the default is 7 days), could instead have the default be "7 days" and use it via `strtotime('7 days', 0);`
- Needs to be documented in README config reference section.

Reply:
Those are minutes not seconds as some like to delete as soon as possible 😉